units/crons.go

```diff
@@ -42,6 +42,9 @@ func PopulateCatchupJobs() amboy.QueueOperation {
 			})
 			return nil
 		}
+		if flags.RepoPollerDisabled {
+			return nil
```

Comment:
Could you add a log message here, similar to the one above for `flags.RepotrackerDisabled`?

Reply:
I thought about that, but we do intend to disable this right away and keep it disabled for a while, so it's going to be logging pretty frequently, whereas we'd only disable the repotracker overall in a crisis. I'm not sure this log would be valuable.